SW_Flow_lib.c

```diff
@@ -1124,22 +1135,21 @@ void hydraulic_redistribution(double swc[], double swcwp[], double lyrRootCo[],
 	}
 }
 
- /**
-     @brief Interpolate soil temperature layer temperature values to
+/**
+@brief Interpolate soil temperature layer temperature values to
      input soil profile depths/layers.
 
-     @param cor[MAX_ST_RGR][MAX_LAYERS + 1]
-     @param nlyrTemp The number of soil temperature layers.
-     @param depth_Temp Depths of soil temperature layers (cm).
-     @param sTempR Temperature values of soil temperature layers (C).
-     @param nlyrSoil The number of soil layer.
-     @param depth_Soil Depths of soil layers (cm).
-     @param width_Soil Witdths of soil layers (cm).
-     @param sTemp Temperatature values soil layers (C).
+@param cor[MAX_ST_RGR][MAX_LAYERS + 1] Two dimensional array containing soiltemperature data.
+@param nlyrTemp The number of soil temperature layers.
+@param depth_Temp Depths of soil temperature layers (cm).
+@param sTempR Temperature values of soil temperature layers (C).

```

Comment:
See if you can add a degree sign, or notation so that a degree sign appears on .html